### ISSUE TITLE
FFMpeg video decode quality setting is ingored

### ISSUE THREAD

**Issue by dustinkerstein** [ultramango/gear360pano#48]
FFMPEGQUALITYDEC doesn't seem to be used on line 330 of gear360video.cmd resulting in very low quality jpeg extracted frames.

**Comment by ultramango** [comment]
Hi!

Thanks for the report, I'll try to reproduce this on my side.

L.

**Comment by ultramango** [comment]
Are we talking about ```-q:v 2``` option? It works for me... have a look at this test code: https://gist.github.com/ultramango/5aefbc0655c530d8e971d4efc4ecc5ea - it will produce ```test_result.csv``` file showing relation between quality parameter and file size.

Could you provide more details? Try running the test script from the link. The only explanation I have is that your ```ffmpeg``` is built with "weird" options, but that's highly unlikely.

More plausible is that the source material itself is low quality. This can happen for quickly changing scenes or scenes with lots of details.

**Comment by dustinkerstein** [comment]
Yeah, I can still replicate this with a fresh ffmpeg build and the latest gear360 repo. It's not the output h264 quality setting, but the temporary jpeg frames that are being rendered at very low quality.

`"%FFMPEGPATH%/ffmpeg.exe" -y -i %1 %FRAMESTEMPDIR%/%IMAGETMPLDEC%`

**Comment by allstone** [comment]
yes, its low quality. Script just does not use higher quality setting. I have put it there manually and then it works.
